package.json

```diff
@@ -38,6 +38,7 @@
     "tricks": "^1.13.5"
   },
   "devDependencies": {
+		"@5app/prettier-config": "^1.0.4",
```

Comment:
This malformed indentation suggests prettier isn't being automatically applied upon commit. For this we'll need to setup something like husky and lint-staged (otherwise pretty-quick)